package.json

```diff
@@ -2,32 +2,30 @@
   "name": "@tableland/sqlparser",
   "version": "0.0.0",
   "files": [
-    "cjs/*.js",
-    "cjs/*.wasm",
-    "cjs/package.json",
+    "*.cjs",
     "*.js",
     "*.d.ts",
     "*.wasm"
   ],
   "exports": {
-    "node": "./cjs/main.js",
+    "node": "./out.cjs",
     "import": "./module.js",
-    "require": "./cjs/main.js",
+    "require": "./out.cjs",
     "default": "./module.js"
   },
   "type": "module",
-  "main": "./cjs/main.js",
+  "main": "./out.cjs",
   "module": "./module.js",
   "types": "types.d.ts",
   "devDependencies": {
     "@types/assert": "^1.5.6",
     "@types/mocha": "^10.0.1",
-    "@types/node": "^18.16.3",
-    "@typescript-eslint/parser": "^5.59.2",
     "@types/node": "^20.4.0",
     "@typescript-eslint/eslint-plugin": "^5.61.0",
     "@typescript-eslint/parser": "^5.61.0",
```

Comment:
This was weird with the duplicate deps. Cleaned it up, but let me know if there is some justification for this.